tools/stern/test/integration/API.hs

```diff
@@ -103,6 +105,34 @@ tests s =
       -- - `POST /teams/:tid/billing`
     ]
 
+testRudSsoDomainRedirect :: TestM ()
+testRudSsoDomainRedirect = do
+  testGet 1 Nothing
```

Comment:
The `Int` parameter does not make a lot of sense to me. I'd rather have something more expressive, but this is not really important.

Reply:
yeah, bad habit of mine.  it's just to locate the failure from the output easier, there isn't much more meaning behind it.